jarbas/core/tests/test_reimbursement_model.py

```diff
@@ -89,6 +89,17 @@ def test_suspicions(self):
         Reimbursement.objects.create(**self.data)
         self.assertEqual(1, Reimbursement.objects.suspicions().count())
 
+    def test_in_latest_dataset(self):
```

Comment:
I would split this test in two: One for the deleted and one for the not-deleted. Even though they are part of the same logical thinking of testing whether or not the reimbursement is available in the latest dataset, it seems to me that breaking this test in two would make easier to detect code failure in a near future.